### ISSUE TITLE
Dokka Documentation

### ISSUE THREAD

**Issue by Doomsdayrs** [korlibs/korge-luak#5]
Using Dokka documentation can be pushed to github pages.

**Comment by soywiz** [comment]
Fair enough. I believe it should be possible to configure gradle, and configure github actions to publish the documentation to pages.

So you know of another repository doing that so I can use it as reference?

**Comment by Doomsdayrs** [comment]
> So you know of another repository doing that so I can use it as reference?

Shosetsu's internal library uses dokka https://gitlab.com/shosetsuorg/kotlin-lib

And here is the [Dokka documentation](https://github.com/Kotlin/dokka)
